vizro-core/src/vizro/managers/_model_manager.py

```diff
@@ -84,28 +84,25 @@ def _get_models(
     def __get_model_children(self, model: Model) -> Generator[Model, None, None]:
         """Iterates through children of `model`.
 
-        Currently looks only through certain fields so might miss some children models.
+        Currently, this method looks only through certain fields so might miss some children models.
```

Comment:
```suggestion
        Currently, this method looks only through certain fields so might miss some children models.
```

What does that mean that it looks "through certain fields"? I thought it's independent of "components", "tabs", "controls", "actions", "selector" now. Or does it refer to something else?

Reply:
I applied your code suggestion @antonymilne  and now it also works with the arbitrary deep `lists` or `dicts`. 🚀